src/utils/obsidian/map-cursor-position-to-idx/map-cursor-position-to-idx.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { describe, expect, it } from "vitest";
-import { mapCursorPositionToIdx } from "./map-position-to-idx.js";
+import { mapCursorPositionToIdx } from "./map-cursor-position-to-idx";
```

Comment:
1) Made file name aligned with the function name
2) Moved both function and test modules into a separate folder